spec/Observable-spec.ts

```diff
@@ -862,24 +862,6 @@ describe('Observable', () => {
     });
   });
 
-  it('should not swallow internal errors', (done) => {
```

Comment:
I don't think this test should be removed?

Reply:
this is a duplicated test from here https://github.com/ReactiveX/rxjs/pull/6436/files#diff-554ac4d3a829f8491d378d79ce9937ca42984e12d35b9b2ee454dca704f00d83R191